src/Models/ProjectLabels.coffee

```diff
@@ -0,0 +1,31 @@
+BaseModel = require '../BaseModel'
+Utils = require '../Utils'
+
+class ProjectLabels extends BaseModel
+  list: (projectId, params = {}, fn = null) =>
```

Comment:
In *ProjectIssues* this method is also called `list()`, should I change this to `all()`?